aten/src/ATen/native/native_functions.yaml

```diff
@@ -1772,8 +1774,7 @@
     SparseCPU: hspmm_sparse_cpu
     SparseCUDA: hspmm_sparse_cuda
 
-# This "raw copy" doesn't handle conversions NOR does it handle non-blocking.
-- func: raw_copy_sparse_(Tensor self, Tensor src) -> Tensor
+- func: copy_sparse_to_sparse_(Tensor self, Tensor src, bool non_blocking=false) -> Tensor
```

Comment:
I'm confused -- where is the implementation for this?

Reply:
It's here: https://github.com/weiyangfb/pytorch/blob/90dea26d6903c401280b443be9f33a36da2d27c1/aten/src/ATen/native/sparse/SparseTensor.cpp#L247-L255

Reply:
wait, I think not, but I define something like: https://github.com/weiyangfb/pytorch/blob/90dea26d6903c401280b443be9f33a36da2d27c1/aten/src/ATen/native/native_functions.yaml#L1777-L1781

maybe it should just work?

Reply:
Gotcha, sorry for missing that line. Yes that would do it